hvplot/ui.py

```diff
@@ -204,22 +204,26 @@ class Geo(Controls):
         can be selected by name or a tiles object or class can be passed,
         the default is 'Wikipedia'.""")
 
-    @param.depends('geo', 'project', 'features', watch=True, on_init=True)
-    def _update_crs(self):
-        enabled = bool(self.geo or self.project or self.features)
-        self.param.crs.constant = not enabled
-        self.param.crs_kwargs.constant = not enabled
-        self.geo = enabled
-        if not enabled:
-            return
+    @param.depends('geo', watch=True, on_init=True)
+    def _update_params(self):
+        enabled = bool(self.geo)
+        geo_controls = set(self.param) - set(Controls.param) - {"geo"}
```

Comment:
This will be cleaner if it is just a list with the parameters. I will change it.